README.md

```diff
@@ -23,3 +23,34 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
+
+| left       | operator | right    | result     | note              |
+| ---------- | -------- | -------- | ---------- | ----------------- |
+| Time       | -        | Time     | Duration   |                   |
+| Time       | +=       | Time     | Time       | seems unexpected? |
```

Comment:
Yes, seems unexpected. What are we adding? If we're silently convert the second time to duration from epoch to that point in time -- I would prefer forcing the user to do this conversion explicitly.

Reply:
Now I remember: this actually already existed (I just moved it). So we might see some breakage somewhere, but should indeed be done explicitly.